src/backend/core/entitlements/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+Entitlements backend utilities.
+"""
+
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+
+def get_entitlements_backend():
```

Comment:
You hsould use the `@cache` annotation here. Once created you should retrun always the same backend, you don't have to instantiate it again and again.